src/common/workspace-store.ts

```diff
@@ -83,6 +83,13 @@ export class Workspace implements WorkspaceModel, WorkspaceState {
     }
   }
 
+  /**
+   * Is this workspace not manged by an extension or is it managed and explicitly enabled
+   */
+  get isEnabled(): boolean {
```

Comment:
Why different name? Should we use `get enabled` and `set enabled` so that api stays the same?

Reply:
I left the `@observable enabled: boolean` as the current method of setting this. Having an `is***` getter seemed to me to be more in line with how our other getters are named. But I can change it if you want.